sandbox.go

```diff
@@ -360,6 +317,12 @@ func (sb *sandbox) Endpoints() []Endpoint {
 	return endpoints
 }
 
+const (
```

Comment:
This is used by other functions in this file. I could have move this aswell but wanted to keep the change minimal. I accidentally moved it above getConnectedEndpoints